library/std/src/keyword_docs.rs

```diff
@@ -16,17 +16,21 @@
 ///
 /// In general, any cast that can be performed via ascribing the type can also be done using `as`,
 /// so instead of writing `let x: u32 = 123`, you can write `let x = 123 as u32` (Note: `let x: u32
-/// = 123` would be best in that situation). The same is not true in the other direction, however,
-/// explicitly using `as` allows a few more coercions that aren't allowed implicitly, such as
-/// changing the type of a raw pointer or turning closures into raw pointers.
+/// = 123` or `let x = 123u32` would be best in that situation). The same is not true in the other
+/// direction, however; explicitly using `as` allows a few more coercions that aren't allowed
+/// implicitly, such as changing the type of a raw pointer or turning closures into raw pointers.
 ///
-/// Other places `as` is used include as extra syntax for [`crate`] and `use`, to change the name
-/// something is imported as.
```

Comment:
I'm not sure what this meant when it said "extra syntax for `crate`", so I removed it. Let me know if it's actually referring to something!

Reply:
It means `extern crate serde as serde2;` and that sort of thing. Here's an example of what it would be useful for: https://github.com/rust-lang/rust/issues/73473

Reply:
Ah, makes sense. Updated.